package.json

```diff
@@ -11,17 +11,17 @@
     "installDeps": "(cd src/zeeguu-react/ && npm install) && ln -s src/zeeguu-react/node_modules .",
     "start": "react-scripts start",
     "prebuild": "rimraf build",
-    "build": "cp ./public/manifest.chrome.json ./public/manifest.json && npm-run-all buildUnix:* && npm-run-all build:*",
-    "buildWindows": "copy .\\public\\manifest.chrome.json .\\public\\manifest.json && npm-run-all buildWindows:* && npm-run-all build:*",
+    "build": "npm-run-all buildUnix:* && npm-run-all build:* && cp ./public/manifest.chrome.json ./build/manifest.json",
+    "buildWindows": "copy .\\public\\manifest.chrome.json .\\build\\manifest.json && npm-run-all buildWindows:* && npm-run-all build:*",
     "buildUnix:popup": "INLINE_RUNTIME_CHUNK=false DISABLE_ESLINT_PLUGIN=true react-scripts build",
     "buildWindows:popup": "SET INLINE_RUNTIME_CHUNK=false && SET DISABLE_ESLINT_PLUGIN=true && react-scripts build",
     "build:content": "webpack --mode production ./src/JSInjection/main.js --output-path ./build/",
     "build:bg": "webpack --mode production ./src/background/background.js --output-path ./build/ --output-filename background.js",
     "build:izo": "webpack --mode production ./src/injectOnZeeguuOrg/injectOnZeeguuOrg.js --output-path ./build/ --output-filename injectOnZeeguuOrg.js",
     "pub": "zip -r build.zip build && open https://chrome.google.com/webstore/devconsole/7f6da896-c12c-4cb3-8ad7-b1641fc8efbc/ckncjmaednfephhbpeookmknhmjjodcd/edit/package",
-    "buildFirefox": "cp ./public/manifest.firefox.json ./public/manifest.json && npm-run-all build:*",
+    "buildFirefox": "npm-run-all build:* && cp ./public/manifest.firefox.json ./build/manifest.json",
```

Comment:
We need to update this to run the buildUnix:popup just like we do in the chrome version.